src/lib/BUILD.bazel

```diff
@@ -0,0 +1,32 @@
+package(default_visibility=["//visibility:public"])
+load("@build_bazel_rules_typescript//:defs.bzl", "ts_library", "ts_config")
+load("@angular//:index.bzl", "ng_package")
+load("//:packages.bzl", "LAYOUT_PACKAGES")
+load("//src/lib:rollup_globals.bzl", "LAYOUT_ROLLUP_GLOBALS")
+
+
+# Export the Layout tsconfig so that subpackages can reference it directly.
+exports_files(["tsconfig-build.json"])
+
+# Root "@angular/flex-layout" entry-point that does not re-export individual entry-points.
+ts_library(
+  name = "flex-layout",
+  srcs = glob(["*.ts"], exclude=["**/*.spec.ts"]),
+  module_name = "@angular/flex-layout",
+  deps = [
+    "//src/lib/core",
+    "//src/lib/extended",
+    "//src/lib/flex",
+    "//src/lib/server",
+  ],
+  tsconfig = ":tsconfig-build.json",
+)
+
+# Creates the @angular/flex-layout package published to npm.
```

Comment:
Should probably add a comment explaining how to publish a release (or point to docs that say `bazel run //src/lib:npm_package.publish` )

Reply:
Added

Reply:
Added link to Bazel.md documentation also.